whois-nrtm/src/test/java/net/ripe/db/whois/nrtm/NrtmConnectionPerIpLimitHandlerTest.java

```diff
@@ -11,14 +13,14 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.ArgumentMatcher;
+import org.mockito.Matchers;
 import org.mockito.Mock;
 import org.mockito.runners.MockitoJUnitRunner;
 
 import java.net.Inet4Address;
 import java.net.InetSocketAddress;
 
-import static org.mockito.Matchers.anyObject;
-import static org.mockito.Matchers.argThat;
+import static org.mockito.Matchers.*;
```

Comment:
Please fix your IDE settings not to use "star" for imports, it's harder to diff changes